Gems/Atom/RPI/Code/Include/Atom/RPI.Public/Culling.h

```diff
@@ -257,10 +257,10 @@ namespace AZ
             void SetOcclusionPlanes(const OcclusionPlaneVector& occlusionPlanes) { m_occlusionPlanes = occlusionPlanes; }
 
             //! Notifies the CullingScene that culling will begin for this frame.
-            void BeginCulling(const AZStd::vector<ViewPtr>& views);
+            void BeginCulling(const Scene& scene, const AZStd::vector<ViewPtr>& views);
 
             //! Notifies the CullingScene that the culling is done for this frame.
-            void EndCulling();
+            void EndCulling(const Scene& scene, const AZStd::vector<ViewPtr>& views);
```

Comment:
repeat: The vector parameter can be made a span
```suggestion
            void EndCulling(const Scene& scene, AZStd::span<const ViewPtr> views);
```